app/controllers/api/firmware_registries_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  class FirmwareRegistriesController < BaseController
+    def create_resource(type, _id, data)
+      assert_id_not_specified(data, type)
+      collection_class(type).create_firmware_registry(data.symbolize_keys)
+    end
+
+    def sync_fw_binaries_resource(type, id, _data)
+      resource_search(id, type, collection_class(type)).sync_fw_binaries_queue
+      action_result(true)
```

Comment:
for action_result signatures, it is preferred to have the same action_result for both success and failures, also should include a message with the message.

i.e.

```
def sync_fw_binaries_resource(type, id, _data)
   do stuff
   action_result(true, "fw_binaries id: #{id} synced")
rescue => err
   action_result(false, err.to_s)
end
```